packages/launch/src/focused-launch/plan-details/index.tsx

```diff
@@ -48,7 +48,7 @@ const PlanDetails: React.FunctionComponent = () => {
 				<Title tagName="h2">{ __( 'Select a plan', __i18n_text_domain__ ) }</Title>
 				<SubTitle tagName="h3">
 					{ __(
-						"There's no risk, you can cancel for a full refund within 30 days.",
+						"There's no risk, you can cancel for a full refund within 14 days.",
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.